### PR TITLE
Fixing Node.js sample issues

### PR DESCRIPTION
**Pull request by munyirik**
- Fixes organization and bugs in the node.js samples
- Adds new "console" filter

### CODE REVIEW

**Comment by munyirik** [comment]
@stephenhuang @ivorb please CR and merge asap to fix the node.js bugs. Thanks!

File: _data/_samples.json [hunk 1]
```diff
@@ -105,13 +106,11 @@
       "title": "'Hello, blinky!'",
       "description": "A 'Hello, world' sample to control an LED from a breadboard",
       "link": [
-        "samples/HelloBlinky.htm",
-        "samples/HelloBlinkyNode.htm"
+        "samples/HelloBlinky.htm"
```
Comment (ivorb) [inline _data/_samples.json]: If we're going to a single link/language per sample, please remove them from the json array - see other examples in this file like Alljoyn.js

File: _data/_samples.json [hunk 2]
```diff
@@ -332,15 +331,13 @@
     },
     {
       "title": "Cylon servo controller",
-      "description": "Control a servo to control a blinking LED",
+      "description": "Control a servo connected to an Arduino using Cylon in a Node.js UWP app",
       "link": [
-        "Samples/CylonServoNode.htm",
-        "Samples/CylonServoWiring.htm"
+        "Samples/CylonServoNode.htm"
```
Comment (ivorb) [inline _data/_samples.json]: same comment as above for this item

File: _data/_samples.json [hunk 3]
```diff
@@ -349,6 +346,23 @@
       ]
     },
     {
+      "title": "Cylon LED controller",
+      "description": "Blink an Arduino LED using Cylon in a Node.js console app",
+      "link": [
```
Comment (ivorb) [inline _data/_samples.json]: again here

File: _data/_samples.json [hunk 4]
```diff
@@ -786,10 +800,28 @@
...
         "intermediate",
+        "console",
         "o/s"
       ]
     },
-
+    {
+      "title": "'Hello, blinky!' console app",
+      "description": "Create a 'Hello, world' console app to control an LED",
+      "link": [
```
Comment (ivorb) [inline _data/_samples.json]: and here

File: en-US/Samples/CylonLEDNode.md [hunk 5]
```diff
@@ -1,14 +1,14 @@
 ---
 layout: sample
-title: Cylon servo controller
-description: Use Cylon to blink an LED on an Arduino from a Windows 10 IoT Core device
-keywords: Windows 10 IoT Core, wiring, arduino
-permalink: /en-US/Samples/CylonServoWiring.htm
+title: Cylon LED controller
+description: Blink an Arduino LED using Cylon in a Node.js console app
+keywords: Windows 10 IoT Core, arduino
+permalink: /en-US/Samples/CylonLEDNode.htm
```
Comment (ivorb) [inline en-US/Samples/CylonLEDNode.md:6]: If changing a file name/location and permalink, we require that we put in a client side redirect for the old file - will you please add back in a CylonServoWiring that redirects to this CylonLEDNode?  info on that can be found in the contribute docs, or copy the way we do in en-US/win10/Docs

**Review by ivorb — CHANGES_REQUESTED** [review]

File: en-US/Samples/CylonLEDNode.md (hunk 5 above)
Comment (munyirik) [reply en-US/Samples/CylonLEDNode.md:6]: Ok. Could you point me to a specific sample .md file that does a redirect?

Comment (ivorb) [reply en-US/Samples/CylonLEDNode.md:6]: Yes, the en-US/win10/Docs.md was meant to be a specific example - just forgot the extension!

Comment (munyirik) [reply en-US/Samples/CylonLEDNode.md:6]: Thanks. So for the permalink I can put "/en-US/Samples/CylonLEDNode.htm" but what should the value of url be?

Comment (munyirik) [reply en-US/Samples/CylonLEDNode.md:6]: Nevermind. Fixed it.

**Review by ivorb — APPROVED** [review]

**Pull request merged**